loadgen/README_BUILD.md

```diff
@@ -0,0 +1,166 @@
+# Building the LoadGen {#ReadmeBuild}
+
+## Quick Start
+
+    pip install absl-py numpy
+    git clone --recurse-submodules https://github.com/mlperf/inference.git mlperf_inference
+    cd mlperf_inference/loadgen
+    CFLAGS="-std=c++14" python setup.py bdist_wheel
```

Comment:
`CXXFLAGS`?

Reply:
How about adding `-O2` or `-O3`?

Reply:
> `CXXFLAGS`?

That didn't work for me for some reason. CFLAGS does.

> How about adding -O2 or -O3?

Done. Added -O3.

Reply:
@psyhtest anything else we need from Brian?

Reply:
Oh by that I mean just for this push :)

Reply:
That's an important clarification! :) No, I think it's great.